contrib/charts/cert-manager/Chart.yaml

```diff
@@ -1,4 +1,12 @@
-apiVersion: v1
-description: A Helm chart for Kubernetes
 name: cert-manager
 version: 0.1.0
```

Comment:
Although we've not properly bumped versions for the chart previously, we definitely should (#223). Maybe this should be 0.2.0? We have made a breaking change to the values.yaml file.